File: Codes/2710-remove-trailing-zeros-from-a-string.py
'''
Author: LetMeFly
Date: 2024-06-29 17:12:29
LastEditors: LetMeFly
LastEditTime: 2024-06-29 17:13:55
'''
class Solution:
    def removeTrailingZeros(self, num: str) -> str:
        for loc in range(len(num) - 1, -1, -1):
            if num[loc] != '0':
                break
        return num[:loc + 1]

File: Codes/2710-remove-trailing-zeros-from-a-string.go
/*
 * @Author: LetMeFly
 * @Date: 2024-06-29 17:17:36
 * @LastEditors: LetMeFly
 * @LastEditTime: 2024-06-29 17:18:42
 */
package main

func removeTrailingZeros(num string) string {
    loc := len(num) - 1
    for num[loc] == '0' {
        loc--
    }
    return num[0 : loc + 1]
}